### PR TITLE
Issue #57

### PR DESCRIPTION
**Pull request by EChesters**
- Added blog to navbars
- Changed Medium icon link to new URL
- Fixed uppercase on navbar to lowercase

screen shot 2016-04-21 at 10 42 31

screen shot 2016-04-21 at 10 43 08

### CODE REVIEW

File: src/css/screen.css [hunk 1]
```diff
@@ -2547,7 +2547,6 @@ nav {
       nav ul li a {
         text-decoration: none;
         font-family: "Lato", sans-serif;
-        text-transform: uppercase;
```
Comment (tanyapowell) [inline src/css/screen.css:2550]: Why was the nav bar text changed from all uppercase?

File: src/newsletter.html [hunk 2]
```diff
@@ -48,7 +48,10 @@
               <a href="projects.html">Projects</a>
             </li>
             <li>
-              <a href="events.html">Events</a>
+              <a href="events.html" class="menu-item">Events</a>
```
Comment (tanyapowell) [inline src/newsletter.html:51]: Nice spot 👍

File: src/sass/partials/_nav.scss [hunk 3]
```diff
@@ -25,7 +25,6 @@ nav {
 			a {
 				text-decoration: none;
 				font-family: $base-font-name;
-				text-transform: uppercase;
```
Comment (tanyapowell) [inline src/sass/partials/_nav.scss:28]: Why was the nav bar text changed from all uppercase?

**Comment by tanyapowell** [comment]
👍 👯

File: src/sass/partials/_nav.scss (hunk 3 above)
Comment (EChesters) [inline src/sass/partials/_nav.scss:28]: It's harder to read for dyslexic people, and screenreaders will read things like 'CONTACT US' like we're offering the opportunity to contact the United States.

**Pull request merged**